doc/admin-guide/monitoring/statistics/core/eventloop.en.rst

```diff
@@ -0,0 +1,146 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+   agreements.  See the NOTICE file distributed with this work for additional information regarding
+   copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with the License.  You may obtain
+   a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software distributed under the License
+   is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+   or implied.  See the License for the specific language governing permissions and limitations
+   under the License.
+
+.. include:: ../../../../common.defs
+
+.. _admin-stats-core-eventloop:
+
+Event Loop
+**********
+
+The core of network processing in |TS| is the :term:`event loop`. This loop executes any pending
+events and performs I/O operations on network connections. The time spent in the event loop is a
+citical component of latency as any network connection has at most one I/O operation per loop. The
+general mechanism is
+
+*  Dispatch any pending events.
+
+*  Check for any pending I/O activity. This wait a variable amount of time. It is at most
+   :ts:cv:`proxy.config.thread.max_heartbeat_mseconds` milliseconds. It is reduced to the amount of
+   time until the next scheduled event. Although this is done in milliseconds, system timers are
+   rarely that accurate.
+
+*  For each network connection dispatch an event to the corresponding network virtual connection
+   object.
+
+*  Dispatch any events generated while handling I/O in the previous step
+
+If event loops are taking a long time this will have a noticeable impact on transaction latency.
+There are a number of statistics gathered to help determine if this is the problem. Because
+instaneous values are not useful, the data is gathered in three different time buckets - 10 seconds,
+100 seconds, and 1000 seconds. There is a parallel set of statistics for each, and each larger time
```

Comment:
how about,
10, 100, and 1000 seconds.
(eg, move the seconds to the end of the list)